### PR TITLE
version(ticdc): adjust cdc version range to 6.3.0-alpha ~ 8.0.0-alpha

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### What problem does this PR solve?

Issue Number: ref #4757 

### What is changed and how it works?

change cdc version from [6.3.0, 8.0.0), to [6.3.0-alpha, 8.0.0-alpha), so that `6.3.0-master` in the range.

#### Tests 

 - Unit test

### Release note 

```release-note
None`
```

### CODE REVIEW

**Review by sdojjy — APPROVED** [review]

File: pkg/version/check_test.go [hunk 1]
```diff
@@ -195,21 +195,22 @@ func TestCheckClusterVersion(t *testing.T) {
...
+	// pre-release version, `vx.y.z-alpha-nightly-yyyymmdd`
+	alphaVersion := semver.New(removeVAndHash("v6.3.0-alpha-nightly-20220202"))
+	require.Equal(t, 1, alphaVersion.Compare(*MinTiCDCVersion))
+
+	// release version, `vx.y.z.`
+	releaseVersion := semver.New(removeVAndHash("v6.3.0"))
+	require.Equal(t, 1, releaseVersion.Compare(*MinTiCDCVersion))
+
+	// build with uncommitted changes, `vx.y.z-dirty`
+	dirtyVersion := semver.New(removeVAndHash("v6.3.0-dirty"))
```
Comment (lonng) [inline pkg/version/check_test.go:211]: Which one is greater between `v6.3.0-dirty` and `v6.3.0`?

**Review by overvenus — APPROVED** [review]
Nice catch

File: pkg/version/check_test.go (hunk 1 above)
Comment (3AceShowHand) [reply pkg/version/check_test.go:211]: they are equal, since `v6.3.0-dirty` will be changed to `v6.3.0` first before comparison.

**Pull request merged**